Sources/Testing/ABI/Encoded/ABI.EncodedIssue.swift

```diff
@@ -51,6 +60,7 @@ extension ABI {
       case .error: .error
       }
       isKnown = issue.isKnown
+      _knownIssueContext = issue.knownIssueContext
```

Comment:
It may be smarter to create a specific `ABI.EncodedKnownIssueContext` type for this, instead of making the actual `Issue.KnownIssueContext` type `Codable`. That's the pattern we have _mostly_ used for the data included in the ABI serialization types; there's a few exceptions but we're trying to move in that direction.

Having this property be underscore-prefixed is a good idea though regardless.

Reply:
Added `ABI.EncodedKnownIssueContext.